src/Uno.UI/UI/Xaml/Extensions/FontHelper.Android.cs

```diff
@@ -113,6 +118,8 @@ private static bool TryLoadFromPath(TypefaceStyle style, string source, out Type
 				? AndroidResourceNameEncoder.EncodeFileSystemPath(source, prefix: "")
 				: source;
 
+			encodedSource = TryAdjustFromManifest(encodedSource, encodePath, style, weight, stretch);
```

Comment:
We shouldn't use manifest for Android as we'll be relying on variable fonts support brought by https://github.com/unoplatform/uno/pull/17309